services/my_lambda/src/my_lambda/stuff/data.csv

```diff
@@ -0,0 +1 @@
+butler:lurch:tall:calm
```

Comment:
I don't see this used in the example.  I know that technically *.delimited files can use a variety of characters to delimit the columns, but given that this file is specifically labeled with a .csv extension, I find it a bit interesting to have a colon for the separator.

Is there a specific reason you chose to use a colon instead of a comma here?

Reply:
Shouldn't be colon, was looking at YAML too much. Fixed to comma. I didn't use it, just wanted to show there could be multiple non code files of different types